packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_IPOUHarmonicCoarseOperator_def.hpp

```diff
@@ -70,6 +70,7 @@ namespace FROSch {
     {
         FROSCH_TIMER_START_LEVELID(initializeTime,"IPOUHarmonicCoarseOperator::initialize");
         int ret = buildCoarseSpace(dimension,dofsPerNode,nodesMap,dofsMaps,nullSpaceBasis,dirichletBoundaryDofs,nodeList);
+        this->CoarseMap_ = this->assembleCoarseMap();
         this->assembleInterfaceCoarseSpace();
```

Comment:
What is the difference in this coarse map and the assembled coarse map from `InterfaceCoarseSpaces_`?

Reply:
`InterfaceCoarseSpaces_` as a vector; I was not sure which entry I should choose; I have to check that!

Reply:
I checked with `InterfaceCoarseSpaces_[blockID]->getBasisMapUnique()` and the maps are different.

Reply:
The vector of interface coarse spaces is also being assembled. Can you explain what the difference is?